scripts/test_util.py

```diff
@@ -0,0 +1,176 @@
+import io
+import subprocess
+import sys
+import traceback
+
+global cl
+
+
+class Colorizer(object):
+    RED = "\033[31;1m"
+    GREEN = "\033[32;1m"
+    YELLOW = "\033[33;1m"
+    CYAN = "\033[36;1m"
+    RESET = "\033[0m"
+    NEWLINE = "\n"
+
+    @classmethod
+    def _colorize(cls, string, color):
+        return getattr(cls, color) + string + cls.RESET + cls.NEWLINE
+
+    @classmethod
+    def red(cls, string):
+        return cls._colorize(string, "RED")
+
+    @classmethod
+    def green(cls, string):
+        return cls._colorize(string, "GREEN")
+
+    @classmethod
+    def yellow(cls, string):
+        return cls._colorize(string, "YELLOW")
+
+    @classmethod
+    def cyan(cls, string):
+        return cls._colorize(string, "CYAN")
+
+
+class FakeStdout(io.StringIO):
+    """Fake class to mimic stdout. We can't just use io.StringIO because we need
+    to fake the ability to write binary files to sys.stdout.buffer (thus this
+    class has a "buffer" attribute that behaves the same way).
+    """
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.buffer = io.BytesIO()
+
+    def getvalue(self):
+        """
+        If self.buffer has a non-unicode value, return that value.
+        Otherwise, decode the self.buffer value and append it
+        to self.getvalue().
+
+        This is because this function is mimicking the behavior of `sys.stdout`.
+        `sys.stdout` can be read as either a string or bytes.
+
+        When a string is written to `sys.stdout`, it returns a string when doing `getvalue()`.
+
+        When bytes are written to `sys.stdout` (by writing to `sys.stdout.buffer`),
+        it returns bytes when doing `getvalue()`. The reason we need to account for this
+        case is that there are tests in which a binary file is uploaded, then it is
+        printed out (by writing to `sys.stdout.buffer`), and then the test reads what's
+        printed out and makes sure it matches the original file.
+        """
+        try:
+            buffer_value = self.buffer.getvalue().decode()
+        except UnicodeDecodeError:
+            return self.buffer.getvalue()
+        return super().getvalue() + buffer_value
+
+
+def run_command(
+    args,
+    expected_exit_code=0,
+    max_output_chars=1024,
+    env=None,
+    include_stderr=False,
+    binary=False,
+    use_cli_directly=False,
```

Comment:
This name is not intuitive to me since CLI can mean either invoking `cl` or creating `BundleCLI`.  Why change it from `force_subprocess`?

Reply:
@percyliang  I will change it back before merging.